PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -266,7 +264,12 @@ def nanoAOD_runMETfixEE2017(process,isData):
     process.nanoSequenceCommon.insert(process.nanoSequenceCommon.index(jetSequence),process.fullPatMetSequenceFixEE2017)
 
 def nanoAOD_customizeCommon(process):
-#    makePuppiesFromMiniAOD(process,True) # call this here as it calls switchOnVIDPhotonIdProducer
+    makePuppiesFromMiniAOD(process,True) # call this here as it calls switchOnVIDPhotonIdProducer
+    process.puppiNoLep.useExistingWeights = True
+    process.puppi.useExistingWeights = True
+    for modifier in run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_94X2016, run2_nanoAOD_102Xv1, run2_nanoAOD_106Xv1:
```

Comment:
My take is that we only want to recompute PUPPI when running the master with run2_nanoAOD_106Xv1 or 10_6_X with run2_miniAOD_UL.
For "run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_94X2016, run2_nanoAOD_102Xv1" we don't want to change the tune, so it should not be modified (and kept True).
Do I misunderstand?

Reply:
Yes, so the correct snippet is:
process.puppiNoLep.useExistingWeights = True
process.puppi.useExistingWeights = True
for modifier in run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_94X2016, run2_nanoAOD_102Xv1:
    process.puppiNoLep.useExistingWeights = True
    process.puppi.useExistingWeights = True
for modifier in run2_nanoAOD_106Xv1:
    process.puppiNoLep.useExistingWeights = False
    process.puppi.useExistingWeights = False

@mariadalfonso @ahinzmann do we all agree now?

Reply:
Hello @mariadalfonso @ahinzmann , please let me know if the version mentioned by @lathomas is okay (although I am not sure if there is a benefit of having the lines with modifers and setting the useExistingWeights = True again, perhaps the following is sufficient? :
makePuppiesFromMiniAOD(process,True)
process.puppiNoLep.useExistingWeights = True
process.puppi.useExistingWeights = True
for modifier in run2_nanoAOD_106Xv1:
    process.puppiNoLep.useExistingWeights = False 
    process.puppi.useExistingWeights = False

Reply:
fine with me. 
This will give the V11 the final tune for the EOY for both Puppi and Taggers compatible with the old tune

Reply:
Done with latest commit